app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="standard_padding">16dp</dimen>
+    <dimen name="standard_margin">16dp</dimen>
+
+    <!-- Increments of 8 for standard spacing -->
+    <dimen name="one">8dp</dimen>
+    <dimen name="two">8dp</dimen>
+    <dimen name="three">8dp</dimen>
+    <dimen name="four">8dp</dimen>
```

Comment:
Whoops, these should be multiples of 8. Fixed in a separate commit here: https://github.com/GWU-CSCI-4237-Fall-2018/Android-Tweets/commit/97f85d1dbd1e7dec5d36534c939f283c3285d2fd